CONTRIBUTING.md

```diff
@@ -1,22 +1,38 @@
-## Your new git-extra command should support
+# Contributing
 
-* OSX, Linux, BSD (You may need to browse their man page)<sup>*</sup>
-* Bash 3.2+ (If you aren't sure, see [the Bash changelog](https://www.tldp.org/LDP/abs/html/bash2.html))
-* Git 2.1+
+Thanks for contributing! Please read this document before you make a PR.
 
-<sup>*</sup>If you aren't able to test your new command on a platform,
-make that clear in your PR and someone else may be able to test it on their system.
+## Supported Platforms
 
-## To submit a new command, you should
+Any changes must support the following platforms:
 
-Let's assume your new command is named `foo`.
+- macOS, Linux, BSDs (You may need to browse their man page)
+- Bash 3.2+ (If you aren't sure, see the [Bash changelog](https://git.savannah.gnu.org/cgit/bash.git/tree/NEWS?h=devel) to see a detailed list of version support for features)
+- Git 2.1+
```

Comment:
Maybe it is time to update the git version?

Reply:
I'll update it to Git 2.17.1, which is a less ancient Git version and is the lowest common denominator between Ubuntu 18 and Debian 10. I'm not sure if it is a good idea to increase it further than that, because I don't know which versions we truly support? Would we have to check all our uses of `git` to find out, or do you have an idea yourself?

Reply:
2.17 LGTM